include_file.c

```diff
@@ -509,38 +510,37 @@ int print_file_names(void) {
 
   struct incbin_file_data *ifd;
   struct file_name_info *fni;
-
+  struct stringmaptable *smt;
+  int is_first_line = YES;
```

Comment:
Note that I changed the way this method works to simplify the logic about when to add space-newline-tab.